client/src/components/TaskCard/TaskCard.js

```diff
@@ -1,20 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
 import { PropTypes } from "prop-types";
 import SmallActionButton from "../SmallActionButton/SmallActionButton.js";
+import TaskDetails from "../../components/TaskDetails/TaskDetails";
 import styles from "./TaskCard.module.css";
 
 const TaskCard = ({
   title,
   location,
   startDateTime,
   endDateTime,
+  description,
   ongoing,
   finished,
-  showDetailsDialog,
   edittable,
   toggleTaskHandler,
   id,
 }) => {
+  const toggleModal = () => {
```

Comment:
I would rename this to `toggleFeedbackModal`

Reply:
Renamed.